src/gtk/toga_gtk/widgets/button.py

```diff
@@ -1,26 +1,23 @@
 from gi.repository import Gtk
 from travertino.size import at_least
-
 from .base import Widget
+from toga_gtk.libs.utils import gtk_apply_css
 
 
 class Button(Widget):
     def create(self):
         self.native = Gtk.Button()
         self.native.interface = self.interface
-
         self.native.connect('show', lambda event: self.rehint())
         self.native.connect('clicked', self.on_press)
 
     def set_label(self, label):
         self.native.set_label(self.interface.label)
         self.rehint()
 
-    def set_enabled(self, value):
-        self.native.set_sensitive(value)
-
     def set_background_color(self, value):
-        self.interface.factory.not_implemented('Button.set_background_color()')
+        if value:
+            gtk_apply_css(self.native, {"background-color": value})
```

Comment:
Is there a reason that this is an external utility method, rather than a method on the base toga-GTK widget?